test/api/v3/integration/groups/POST-groups_groupId_join.js

```diff
@@ -0,0 +1,51 @@
+import {
+  generateUser,
+} from '../../../../helpers/api-integration.helper';
+
+describe('POST /group/:groupId/join', () => {
+  let user;
+
+  beforeEach(() => {
+    return generateUser().then((generatedUser) => {
+      user = generatedUser;
+    });
+  });
+
+  context('Guilds', () => {
+    it('inviting to guild doesn\'t give basilist to inviter', async () => {

```

Comment:
This should read more naturally in English: `it inviting to guild doesn't give basilist to inviter`

Perhaps:

```
it('does not give basilist quest to inviter when joining a guild', async () => {
```